### PR TITLE
Fixes for JS error and word breaks.

### PR DESCRIPTION
**Pull request by mbrandonw**
Fixes a few problems we have on the episode page right now:

* JS error because of some old JS that dealt with video.js
* Long code lines in corrections and exercises cause the page to overflow horizontally

### CODE REVIEW

File: Sources/Views/Markdown.swift [hunk 1]
```diff
@@ -33,6 +33,7 @@ public let markdownBlockStyles: Stylesheet =
       <> blockquoteMarkdownStyles
       <> pMarkdownStyles
       <> codeMarkdownStyles
+      <> key("word-break", "break-word")
```
Comment (mbrandonw) [inline Sources/Views/Markdown.swift]: decided to just break words on all markdown containers. fixes the problem in a bunch of places at once

File: Sources/PointFree/Episode/Show.swift [hunk 2]
```diff
@@ -34,7 +34,6 @@ let episodeResponse =
           currentUser: currentUser,
           data: (permission, currentUser, subscriberState, episode),
           description: episode.blurb,
-          extraHead: videoJsHead,
```
Comment (stephencelis) [inline Sources/PointFree/Episode/Show.swift]: This'll temporarily making video.js videos less featured, right?

File: Sources/Views/Markdown.swift (hunk 1 above)
Comment (stephencelis) [reply Sources/Views/Markdown.swift]: Nice, thanks for doing this!

File: Sources/PointFree/Episode/Show.swift (hunk 2 above)
Comment (mbrandonw) [reply Sources/PointFree/Episode/Show.swift]: oh yes, I forgot we still have some video.js. I'll bring it back for now

**Pull request merged**